src/main/java/com/passbook/sparkeighteen/service/UserService.java

```diff
@@ -94,11 +95,18 @@ public SignUpResponse signUp(@NonNull final SignUpRequest request) {
 
     }
 
-
     private Integer calculateAge(LocalDate dob) {
         return Period.between(dob, LocalDate.now()).getYears();
     }
 
+    public String deleteProfile(Integer userId) {
```

Comment:
What will happen to profile 
Explain how its deleted?

Reply:
Do you also need to delete user think about it

Reply:
Yes, every time we delete profile, user  should be deleted
Because passbook user should have only one profile.

Reply:
You are deleting the user so the profile is getting deleted is it right?
because of CascadeType.ALL?

Reply:
yes.

Reply:
Yes, that's right.

Reply:
Yes

Reply:
Yes